### ISSUE TITLE
Rack Elevation change of displayed parameters

### ISSUE THREAD

**Issue by martin-networker** [nautobot/nautobot#2336]
### As ...

Ozzie - Operator

### I want ...

To define what information will be displayed above racks in Rack Elevation.

### So that ...

I can see in `Rack Elevation` parameters which are useful for planning and deployment process of devices.
For example I need to see `Tenant `and `Rack Groups` in `Rack Elevation` so I know in which rack I can install device which belongs to a specific `Tenant `and `Rack Groups`.
Currently there are two line above each rack:
   - first displays `Rack name` and `Facility ID`
   - second display again `Facility ID`

### I know this is done when...

I open Rack Elevation :)

### Optional - Feature groups this request pertains to.

- [ ] Automation
- [ ] Circuits
- [X] DCIM
- [ ] IPAM
- [ ] Misc (including Data Sources)
- [ ] Organization
- [ ] Plugins (and other Extensibility)
- [ ] Security (Secrets, etc)
- [ ] Image Management
- [ ] UI/UX
- [ ] Documentation
- [ ] Other (not directly a platform feature)

**Comment by lampwins** [comment]
We will render each attribute as a separate line in the elevation view. Additionally, we should not use the display name in the elevation view because it duplicates the facility ID.

**Comment by martin-networker** [comment]
@lampwins the rack name should be displayed but it should not contain facility ID because it is already displayed below and because it makes the name too long and it overlaps with next rack name and thus make it unreadable - maybe you meant it like this but I just wanted to clarify it :)
Regarding rendering each attribute, it will be good if user could choose which attributed to display as displaying all attributes is not desirable - again, maybe you meant it like this but I just wanted to clarify it :)

**Comment by bryanculver** [comment]
> @lampwins the rack name should be displayed but it should not contain facility ID because it is already displayed below and because it makes the name too long and it overlaps with next rack name and thus make it unreadable - maybe you meant it like this but I just wanted to clarify it :)
> Regarding rendering each attribute, it will be good if user could choose which attributed to display as displaying all attributes is not desirable - again, maybe you meant it like this but I just wanted to clarify it :)

The "display name" is a particular property of the Rack (it's a concatenation of the Rack's name and the facility ID). Instead in the elevation view we will switch to just the rack's name and include additional information below.

Full customizability is an ideal goal but we can incrementally improve this UI without needing to go all that way. Is there any information beyond `Rack Group` and `Tenant` you'd like to see in this view?

**Comment by martin-networker** [comment]
@bryanculver after discussion with our installation team  `Rack Group` and `Tenant ` are the only one we are interested in.
One related question: what do you this about possibility for the `Rack `to be a member of multiple `Rack Group` . For example `Rack Group` 1-Phase and concurrently `Rack Group`  Storage. If you like this idea I will open a new issue for this.

**Comment by glennmatthews** [comment]
> One related question: what do you this about possibility for the `Rack `to be a member of multiple `Rack Group` . For example `Rack Group` 1-Phase and concurrently `Rack Group` Storage. If you like this idea I will open a new issue for this.

To me that sounds like a use case for tags, rather than rack groups.

**Comment by glennmatthews** [comment]
One possibility here might be providing a config option similar to the [`UI_RACK_VIEW_TRUNCATE_FUNCTION`](https://docs.nautobot.com/projects/core/en/stable/user-guide/administration/configuration/optional-settings/#ui_rack_view_truncate_function) to provide full customizability of this particular piece of UI, but that may be more complexity than is strictly needed at this time.

**Comment by martin-networker** [comment]
@glennmatthews Thanks for this update.
Rack Elevatin currently displays above Racks only Name, Role and Facility ID.
We need another parameters (rack_environment, number of phases in rack, Power, number of uplinks, etc.) to be displayed, ideally in same manner as Role (standalone rectangle with its color).
We already use Role so currently our only possibility is to put all parameters into Facility ID as one string. This is not ideal and as Facility ID is automatically added to rack name (showing same parameter twice at the same place) causing names of racks to rewrite each other and making them unreadable.

**Comment by lampwins** [comment]
Would it be acceptable to list all (most) of the rack fields at the bottom of the elevation view?

**Comment by martin-networker** [comment]
@lampwins Yes that would work but this would have to include custom fields as all the parameters we need to see are extra/custom parameters.
